draft-ietf-quic-transport.md

```diff
@@ -960,12 +960,10 @@ Every QUIC packet that is coalesced into a single UDP datagram is separate and
 complete.  Though the values of some fields in the packet header might be
 redundant, no fields are omitted.  The receiver of coalesced QUIC packets MUST
 individually process each QUIC packet and separately acknowledge them, as if
-they were received as the payload of different UDP datagrams.  If one or more
-packets in a datagram cannot be processed yet (because the keys are not yet
-available) or processing fails (decryption failure, unknown type, etc.), the
-receiver MUST still attempt to process the remaining packets.  The skipped
-packets MAY either be discarded or buffered for later processing, just as if the
-packets were received out-of-order in separate datagrams.
+they were received as the payload of separate UDP datagrams.  For example, if
+decryption fails (the keys are not yet available or any other reason) or the
+packet is of an unknown type, the receiver MAY either discard or buffer the
```

Comment:
"the packet" does not make sense in this context. How about "If one or more packets in a datagram cannot be processed yet, because of decryption failure or any other reason, the receiver MAY either discard or buffer the packets for later processing. The receiver MUST still attempt to process the remaining packets in the UDP datagram."

Reply:
That sounds good.  Want to port that over to #1857?

Reply:
I would omit "yet".  "Yet" implies knowing that it will be possible to decrypt the packet in the future.  It is hard to know the future.